### PR TITLE
add issue template

### PR DESCRIPTION
**Pull request by carlory**

### CODE REVIEW

**Comment by carlory** [comment]
cc @wzshiming

**Comment by carlory** [comment]
cc @wzshiming

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      required: true
+
+  - type: textarea
+    id: installer
+    attributes:
+      label: Install tools
+      value: |
+        <details>
+
+        </details>
```
Comment (wzshiming) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:60]:
```suggestion
    id: osVersion
    attributes:
      label: OS version
      value: |
        <details>
        ```console        
        # On Linux:
        $ cat /etc/os-release
        # paste output here
        $ uname -a
        # paste output here

        # On Darwin:
        $ uname -a
        # paste output here

        # On Windows:
        C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
        # paste output here
        ```
        </details>
```

**Review by wzshiming — CHANGES_REQUESTED** [review]
Add a configuration file for the Support Request

.github/ISSUE_TEMPLATE/config.yml
```
contact_links:
  - name: Support Request
    url: https://github.com/kubernetes-sigs/kwok/discussions
    about: Support request or question relating to Kwok
```

**Comment by carlory** [comment]
@wzshiming updated!

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
+name: Bug Report
+description: Report a bug encountered while operating Kwok
+labels: kind/bug
+body:
```
Comment (wzshiming) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:4]:
```suggestion
body:
  - type: checkboxes
    id: usage
    attributes:
      label: How to use it?
      options:
        - label: kwok
        - label: kwokctl --runtime=docker (default runtime)
        - label: kwokctl --runtime=binary
        - label: kwokctl --runtime=nerdctl
        - label: kwokctl --runtime=kind

```

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        </details>
+
+  - type: textarea
+    id: runtime
+    attributes:
+      label: Runtime (binary, docker-compose or kind) and version if (applicable)?
+      value: |
+        <details>
+
+        </details>
```
Comment (wzshiming) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:82]:
```suggestion
```

**Review by wzshiming — COMMENTED** [review]
Maybe should add [PULL_REQUEST_TEMPLATE.md](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md) as well

**Comment by carlory** [comment]
@wzshiming updated!

**Comment by wzshiming** [comment]
```
[*] Verifying ends newline...
Add a new line in ends for below files
./.github/ISSUE_TEMPLATE/enhancement.yaml
./.github/ISSUE_TEMPLATE/failing-test.yaml
```

**Comment by carlory** [comment]
@wzshiming updated

**Pull request merged**